warrior/WarriorCore/Classes/kw_driver_class.py

```diff
@@ -201,7 +201,6 @@ def get_argument_as_keywords(self):
     def execute_method_for_keyword(self):
         """Executes a method corresponding to keyword """
 
-        Utils.config_Utils.set_datarepository(self.data_repository)
```

Comment:
in that case can you remove line 226 under the method execute_function_for_keyword as well